src/django/api/templates/mail/reset_user_password_body.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>
+            Reset your Open Apparel Registry password
+        </title>
+    </head>
+    <body>
+        <p>
+            Hi {{ user.name }},
+        </p>
+        <p>
+            We're sending you this email because someone requested to reset the password
+            for the Open Apparel Registry account associated with this email address.
+        </p>
+        <p>
+            You can reset your password by visiting the following link:
+            <a href="{{ domain }}/auth/resetpassword/{{ uid }}?token={{ token }}">
```

Comment:
We should consider prefixing our links with `https://` so they don't default to `http`.